pkg/flash/flash_linux.go

```diff
@@ -53,36 +46,60 @@ func New(spi SPI) (*Flash, error) {
 	}
 
 	var err error
+	var id chips.ID
+	id, err = f.spi.ID()
+	if err != nil {
+		return nil, fmt.Errorf("Can not ID chip: %w", err)
```

Comment:
errors strings should not be capitalised https://go.dev/wiki/CodeReviewComments#error-strings

Reply:
fixed, thanks.